src/PreCICE.jl

```diff
@@ -931,16 +931,16 @@ coordinates omitting the mapping. This function is still experimental.
 - `vertexIDs::AbstractArray{Integer}`: IDs of the vertices.
 - `vertexCoordinates::AbstractArray{Float64}`: Coordinates of the vertices and corresponding data values. Shape [N x D] where N = number of vertices and D = number of data dimensions.
 """
-function getMeshVerticesAndIDs(
+function getMeshVertexIDsAndCoordinates(
     meshName::String,
 )::Tuple{AbstractArray{Integer},AbstractArray{Float64}}
-    @warn "The function getMeshVerticesAndIDs is still experimental"
+    @warn "The function getMeshVertexIDsAndCoordinates is still experimental"
```

Comment:
The function is not experimental any more (as you know). Same for the setMeshAccessRegion.